tests/test_helpers.py

```
@@ -0,0 +1,75 @@
+
+# Add custom_components to path
+sys.path.insert(0, str(Path(__file__).parent.parent))
+
+from custom_components.acwd.helpers import local_midnight
+
+
+@pytest.mark.unit
+class TestLocalMidnight:
+    """Tests for local_midnight() utility function."""
+
+    def test_returns_timezone_aware_datetime(self, pst_timezone):
```

Ruff (0.15.2)

[warning] Unused method argument: `pst_timezone`

(ARG002)